<ui version="4.0" >
 <class>listChannelClass</class>
 <widget class="QWidget" name="listChannelClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Channels List</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="irc.qrc" >
    <normaloff>:/icons/irc-online.png</normaloff>:/icons/irc-online.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QFrame" name="frame" >
       <property name="minimumSize" >
        <size>
         <width>281</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>281</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="editFilter" >
        <property name="geometry" >
         <rect>
          <x>158</x>
          <y>2</y>
          <width>102</width>
          <height>25</height>
         </rect>
        </property>
        <property name="maximumSize" >
         <size>
          <width>102</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label" >
        <property name="geometry" >
         <rect>
          <x>99</x>
          <y>1</y>
          <width>55</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text" >
         <string>Filter by:</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btnRequest" >
        <property name="geometry" >
         <rect>
          <x>1</x>
          <y>1</y>
          <width>94</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text" >
         <string>Request list</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2" >
       <property name="minimumSize" >
        <size>
         <width>151</width>
         <height>31</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="labelStatus" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>207</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="imgLoad" >
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;img src=":/icons/irc-loading.gif" />&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="table" >
     <column>
      <property name="text" >
       <string>Users</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Channel</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Topic</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="irc.qrc" />
 </resources>
 <connections/>
</ui>
